operations/commit_queue.go

```diff
@@ -394,19 +396,20 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 	if err != nil {
 		return errors.Wrap(err, "can't get commit count")
 	}
-	if commitCount != 1 {
-		return errors.Errorf("patch contains %d commits, must contain 1", commitCount)
+	if commitCount > 1 && !confirm("Commit queue patch has multiple commits. Continue? (y/n):", false) {
```

Comment:
should we stop the user from adding a module if there are 0 commits?

Reply:
Why should this be different for modules? Should we not also do this for regular patches then?

Reply:
For patches there is a case where if a user is trying to make a commit only to a module's repo the original patch will have no commits.

Reply:
okay this is done